### DISCUSSION TITLE
Feature Request: Add `/packages/subgraph/graph-node/data` to .gitignore for subgraph-package

### DISCUSSION THREAD

**Discussion by dentropy** [scaffold-eth/scaffold-eth-2#388] [Ideas]
Feature Request: Add `/packages/subgraph/graph-node/data` to .gitignore for subgraph-package

The postgres container created in the [docker-compose file](https://github.com/scaffold-eth/scaffold-eth-2/blob/subgraph-package/packages/subgraph/graph-node/docker-compose.yml) create a data directory within the project folder, I don't want to have that showing up when I run git commands.

**Comment by dentropy** [comment]
Related Pull Request,

https://github.com/scaffold-eth/scaffold-eth-2/pull/405

**Comment by schmidsi** [comment]
I think this is solved: https://github.com/scaffold-eth/scaffold-eth-2/blob/subgraph-package/packages/subgraph/.gitignore#L1
